website/docs/FAQ.md

```diff
@@ -24,9 +24,9 @@ config_list = autogen.config_list_from_json(
 )
 ```
 
-It first looks for environment variable "OAI_CONFIG_LIST" which needs to be a valid json string. If that variable is not found, it then looks for a json file named "OAI_CONFIG_LIST" under the specified `file_location`. It then filters the configs by models (you can filter by other keys as well).
+It first looks for environment variable "OAI_CONFIG_LIST" which needs to be a valid json string. If that variable is not found, it then looks for a json file named "OAI_CONFIG_LIST" under the specified `file_location` (defaulted to the root of the repo). It then filters the configs by models (you can filter by other keys as well).
```

Comment:
```suggestion
It first looks for environment variable "OAI_CONFIG_LIST" which needs to be a valid json string. If that variable is not found, it then looks for a json file named "OAI_CONFIG_LIST" under the specified `file_location` (defaulted to the working directory). It then filters the configs by models (you can filter by other keys as well).
```

Reply:
I think it's "working directory" rather than "root of the repo" and it should not be the "root of the repo" as if someone pip installed pyautogen, they will not put the file to the repo.